### ISSUE TITLE
make midstate fails on osx

### ISSUE THREAD

**Issue by razorfish-sl** [CryptoManiac/stratum-mining-proxy#2]
make
gcc -march=native -Wall -funroll-all-loops -O3 -fstrict-aliasing -Wall -std=c99 -I/usr/include/python2.7  midstatemodule.c -o test -Wl,-O1 -Wl,--as-needed -lpython2.7
midstatemodule.c:1: error: bad value (native) for -march= switch
midstatemodule.c:1: error: bad value (native) for -mtune= switch
make: **\* [test] Error 1

**Comment by vpereira** [comment]
just as short notice @razorfish-sl. I was able to compile it using clang, my Makefile is here https://gist.github.com/vpereira/6502201

**Comment by CryptoManiac** [comment]
razorfish-sl, can you publish output of gcc --version command?

**Comment by razorfish-sl** [comment]
gcc --version 
i686-apple-darwin11-llvm-gcc-4.2 (GCC) 4.2.1 (Based on Apple Inc. build 5658) (LLVM build 2336.11.00)

**Comment by razorfish-sl** [comment]
Yep  Just thought I would push the error so the author could fix it.
